lib/mbta.js

```diff
@@ -2,10 +2,43 @@
 
 // Code here.
 
-const stopsBetweenStations = (startLine, startStation, endLine, endStation) => {
+// Sets up MBTA lines
+let redLine = ['South Station', 'Park Street', 'Kendall', 'Central'];
+redLine = redLine.concat(['Harvard', 'Porter', 'Davis', 'Alewife']);
+let greenLine = ['Government Center', 'Park Street', 'Boylston', 'Arlington'];
+greenLine = greenLine.concat(['Copley', 'Hynes', 'Kenmore']);
+let orangeLine = ['North Station', 'Haymarket', 'Park Street', 'State'];
+orangeLine = orangeLine.concat(['Downtown Crossing', 'Chinatown']);
+orangeLine = orangeLine.concat(['Back Bay', 'Forest Hills']);
```

Comment:
I was. In later version i just used commas and went across lines that way which was also fine.